.github/workflows/nightly.yml

```diff
@@ -136,12 +205,59 @@ jobs:
         done
         kubectl logs $podname
 
-        # placeholder for now
-        echo "Grafana Link: https://grafana.lab.wlan.tip.build/TO_BE_PROVIDED"
-        echo "Kibana Link: https://kibana.lab.wlan.tip.build/TO_BE_PROVIDED"
-      shell: bash
-    - name: Publish Unit Test Results
+    - name: Upload Unit Test Results
+      if: always()
+      uses: actions/upload-artifact@v2
+      with:
+        name: Unit Test Results ( nola-${{ matrix.number }} )
+        path: /ci/test_everything.xml
```

Comment:
Maybe we should name the file with a `matrix.number` postfix and the date? Not sure where this is uploaded to, however.

Reply:
these are only uploaded temporary, to be consumed by the real upload results task, but I'm not sure and we can probably do what you advice anyway. shall be better if we can actually retrieve them

Reply:
these are only available under the tests they were uploaded at, so tagging them with day probably doesn't make a lot sense? 
![image](https://user-images.githubusercontent.com/16194643/107359731-1870cc80-6ae6-11eb-85ae-c1b7999026cd.png)